### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by patryk-babon**
- [DEMO LINK](https://patryk-babon.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/utils/_mixins.scss
Comment (choeqq) [inline src/styles/utils/_mixins.scss:1]: I would create separate animation for each hand - seconds hand is looking pretty smooth, when the other 2 are just jumping over few pixels :/

**Review by choeqq — CHANGES_REQUESTED** [review]
Almost there 😄

File: src/styles/utils/_mixins.scss
Comment (patryk-babon) [reply src/styles/utils/_mixins.scss:1]: This is a deliberate effect. The hour hand jumps every hour and the minute hand jumps every minute. This is the effect I wanted and I like it.

The animation can be easily customized as you can see in the example below:

```css
@include rotate($seconds-rotation-time, linear);

@include rotate($minutes-rotation-time, steps(60));
```

**Review by choeqq — APPROVED** [review]
I don't have any other adjustments, nicely done! 🚀

**Pull request closed without merging**